lib/linguist/languages.yml

```diff
@@ -1347,7 +1347,7 @@ GDScript:
 GLSL:
   type: programming
   extensions:
-  - ".glsl"
+  - ".comp"
```

Comment:
The first extension doesn't need to be alphabetically sorted. It should be the primary extension of the language.